shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -179,6 +179,22 @@ - (void)collectRoutes:(NSMutableArray<SemanticsObject*>*)edges {
   }
 }
 
+- (NSString*)scrollNotification {
+  int32_t scrollIndex = [self node].scrollIndex;
+  int32_t scrollChildren = [self node].scrollChildren;
+  if (scrollChildren == 0 || ![self hasChildren]) {
+    return @"";
+  }
+  int visibleChildren = 0;
+  for (SemanticsObject* child in self.children) {
+    if (![child node].HasFlag(blink::SemanticsFlags::kIsHidden)) {
+      visibleChildren += 1;
+    }
+  }
+  // TODO: add translations.
+  return [NSString stringWithFormat:@"Showing items %d to %d of %d", scrollIndex + 1, scrollIndex + visibleChildren, scrollChildren];
```

Comment:
There is no API to let iOS generate this basic string like on Android?

Reply:
Not that I could find. I can explore using an out of band message for scroll announcements for iOS, but it seems like for switches we will need an engine localized value anyway